connectivity/check/context.go

```diff
@@ -593,34 +579,51 @@ func (ct *ConnectivityTest) report() error {
 	}
 
 	if ct.params.Perf {
-		if ct.params.PerfLatency {
-			// Report Performance results for latency
-			ct.Header("🔥 Latency Test Summary:")
-			ct.Logf("%s", strings.Repeat("-", 233))
-			ct.Logf("📋 %-15s | %-50s | %-15s | %-15s | %-15s | %-15s | %-15s | %-15s | %-15s | %-15s | %-15s", "Scenario", "Pod", "Test", "Num Samples", "Duration", "Min", "Mean", "Max", "P50", "P90", "P99")
-			ct.Logf("%s", strings.Repeat("-", 233))
-			for p, d := range ct.PerfResults {
-				ct.Logf("📋 %-15s | %-50s | %-15s | %-15d | %-15s | %-12.2f %s | %-12.2f %s | %-12.2f %s | %-12.2f %s | %-12.2f %s | %-12.2f %s",
-					d.Scenario, p.Pod, p.Test, d.Samples, d.Duration,
-					d.Latency["min"][0], d.Metric,
-					d.Latency["mean"][0], d.Metric,
-					d.Latency["max"][0], d.Metric,
-					d.Latency["p50"][0], d.Metric,
-					d.Latency["p90"][0], d.Metric,
-					d.Latency["p99"][0], d.Metric)
+		ct.Header("🔥 Network Performance Test Summary:")
+		ct.Logf("%s", strings.Repeat("-", 200))
```

Comment:
Might be cleaner to define a PerfResult type vs the simple map and hang Printer methods to output the different perf tables. Also you could hang helpers aka nodeString.
We could  perhaps leverage a table printing lib and use it for tabular output.
Lastly we could sort the results to make it for a quick perf scan.

Reply:
I agree that using some kind of table lib would be great.

Reply:
So I definitely agree with using some lib for printing table and also cleaning up this mess, but I think it's out of scope of this PR. I am happy to create follow-up issue for that. 
Re sorting, results are already sorted in `PerfResults` - that's why I changed it from map to list (before they weren't sorted) + added sorting here https://github.com/cilium/cilium-cli/blob/0099fbf06547f6168075702b1e4a9f3ffa1f8104/connectivity/check/deployment.go#L1097
While I would do it differently normally and sort results in the `presentation` layer, I wanted to do it as simply as possible for now as this part was not really my goal of improving netperf command.